<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>checkboxVibration</class>
 <widget class="QDialog" name="checkboxVibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(73, 125, 168)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>512</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/prefijoNuevo/images/game1_config.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>1241</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ELIGE BOTÓN:</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonNode1">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(52, 101, 164)</string>
      </property>
      <property name="text">
       <string>BOTÓN 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonNode2">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(52, 101, 164)</string>
      </property>
      <property name="text">
       <string>BOTÓN 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonNode3">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(52, 101, 164)</string>
      </property>
      <property name="text">
       <string>BOTÓN 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonNode4">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(52, 101, 164)</string>
      </property>
      <property name="text">
       <string>BOTÓN 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonNode5">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(52, 101, 164)</string>
      </property>
      <property name="text">
       <string>BOTÓN 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonNode6">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(52, 101, 164)</string>
      </property>
      <property name="text">
       <string>BOTÓN 6</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="labelColorNode">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>COLOR BOTÓN X:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonColorNode">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="labelBuzzer">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CHICHARRA:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxBuzzer">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(52, 101, 164)</string>
      </property>
      <property name="text">
       <string>ACTIVADO / DESACTIVADO</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>800</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="labelVibration">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>VIBRACIÓN:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxVibration">
      <property name="styleSheet">
       <string notr="true">background-color:rgb(52, 101, 164)</string>
      </property>
      <property name="text">
       <string>ACTIVADO / DESACTIVADO</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>900</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="labelEmotionActual">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EMOCION ACTUAL:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonEmotionActual">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonConfigGame1Save">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>990</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(245, 121, 0)</string>
   </property>
   <property name="text">
    <string>GUARDAR (S)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonConfigGame1Exit">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>990</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(204, 0, 0)</string>
   </property>
   <property name="text">
    <string>VOLVER (V)</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>620</y>
     <width>981</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="labelOptionColors">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>COLORES OPCIONALES:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSetColor1">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSetColor2">
      <property name="styleSheet">
       <string notr="true">background:rgb(78, 154, 6)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSetColor3">
      <property name="styleSheet">
       <string notr="true">background:rgb(252, 233, 79)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSetColor4">
      <property name="styleSheet">
       <string notr="true">background:rgb(32, 74, 135)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSetColor5">
      <property name="styleSheet">
       <string notr="true">background:rgb(143, 89, 2)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSetColor6">
      <property name="styleSheet">
       <string notr="true">background: rgb(245, 121, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSetColor7">
      <property name="styleSheet">
       <string notr="true">background:rgb(92, 53, 102)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonSetColor8">
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 0, 221)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>900</y>
     <width>731</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="labelSelectEmotion">
      <property name="font">
       <font>
        <family>DejaVu Sans</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>SELECCIONA OTRA EMOCÍON:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxEmotions"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="assets/source.qrc"/>
 </resources>
 <connections/>
</ui>
